src/components/Navbar/Navbar.jsx

```diff
@@ -0,0 +1,64 @@
+import "./Navbar.css";
+import CurrentUserContext from "../../contexts/CurrentUserContext";
+import { useContext } from "react";
+import { Link } from "react-router-dom";
+
+function Navbar({
+  handleLoginUser,
+  handleRegisterUser,
+  handleEditProfileUser,
+  handleLogout,
+  isLoggedIn,
+}) {
+  const currentUser = useContext(CurrentUserContext);
+  return (
+    <section className="nav">
```

Comment:
* My remark hasn't been corrected https://github.com/AJRusch/Recipe-Roulette-Final-Project/pull/2#discussion_r1874416015 . Please, use a `nav` tag inside `Navbar`, according to the semantics

`nav` is needed instead of `section`

```jsx
  return (
    <nav className="nav">
      <div className="nav__menu">

```